lib/praxis/field_expander.rb

```diff
@@ -50,6 +60,16 @@ def expand_fields(attributes, fields)
       end
 
       attributes.each_with_object({}) do |(name, dumpable), hash|
+        # Filter out attributes that are not displayable (if a filter is provided and there is a displayable option)
+        if( privs = dumpable.options[:displayable] )
```

Comment:
Rubocop would cry at `if( ... )`

Reply:
actually, it is an assignment, so it would cry without the parens.
BUT, that aside, if I run rubocop the repo is full of warnings...so, what I am going to do, is to merge this PR as such...and then do an big autocorrect in a different PR/commit to get a little more compliant